### ISSUE TITLE
Use 2.1_conditional_generation.py , after fine-tuning, it only generates the same char.  Why ?

### ISSUE THREAD

**Issue by 353xiong** [thunlp/OpenPrompt#140]
use 2.1_conditional_generation.py  in datasets/CondGen/webnlg_2017/

generated txt： ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''

**Comment by ShengdingHu** [comment]
Can you provide more detailed information? For example, what modifications did you make? How many iterations did you run?

**Comment by 353xiong** [comment]
I add this (tokenizer.bos_token = 'bos'), if not add this, it will can't run.
And I use default parameters.  The iterations is 5(default).
The iterations is 5(default) . And I try more(30), the learning rate will drop to 0, the generted txt become :<unk> <unk><unk>

parser = argparse.ArgumentParser("")
parser.add_argument("--lr", type=float, default=5e-5)
parser.add_argument("--plm_eval_mode", action="store_true")
parser.add_argument("--model", type=str, default='t5')  # tested model are gpt2/t5
parser.add_argument("--model_name_or_path", default='t5-base')
args = parser.parse_args()

**Comment by 353xiong** [comment]
> Can you provide more detailed information? For example, what modifications did you make? How many iterations did you run?

why it generated sentences 
.<extra_id_1>.<extra_id_2>.<extra_id_3>. The<extra_id_4>. The<extra_id_5>. The<extra_id_6>. The<extra_id_7>. The<extra_id_8>. The<extra_id_9>. The<extra_id_10>. The<extra_id_11>. The<extra_id_12>. The<extra_id_13>. The<extra_id_14>. The<extra_id_15>. The<extra_id_16>. The<extra_id_17>. The<extra_id_18>. The<extra_id_19>. The<extra_id_20>. The<extra_id_21>. The<extra_id_22>. The<extra_id_23>. The<extra_id_24>. The<extra_id_25>. The<extra_id_26>. The<extra_id_27>.
.<extra_id_1>.<extra_id_2>.<extra_id_3>. The<extra_id_4>. The<extra_id_5>. The<extra_id_6>. The<extra_id_7>. The<extra_id_8>. The<extra_id_9>. The<extra_id_10>. The<extra_id_11>. The<extra_id_12>. The<extra_id_13>. The<extra_id_14>. The<extra_id_15>. The<extra_id_16>. The<extra_id_17>. The<extra_id_18>. The<extra_id_19>. The<extra_id_20>. The<extra_id_21>. The<extra_id_22>. The<extra_id_23>. The<extra_id_24>. The<extra_id_25>. The<extra_id_26>. The<extra_id_27>.

**Comment by ShengdingHu** [comment]
What's your python version? Please use python=3.8 for this example, since python=3.7 has strange bugs about the keyword arguments. See Readme.md for related issues. We will also try to find a workaround recently on python=3.7
